### PR TITLE
feat!: configure semantic versioning, this commit to force version 1.0.0

### LINKED ISSUE #6: Implement semantic versioning for the application
Use semantic versioning for the application and allows version to be autocalculated from the commit.

Suggestion is to use [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) and `com.github.jmongard.git-semver-plugin` in order to compute the version.

When a release tag is pushed, the release job should create the package for all the versions.

### PR DESCRIPTION
**Pull request by sciack**
Implement plugin to calculate semantic versining and propagate to the subproject.

Not sure how to do it with iOS, maybe is needed some processing on XML or set some environment variable.

**NOTE:** Check the readme when application name change in iOS, a cleanup is needed and a reimport of the project.

### CODE REVIEW

File: iosApp/Configuration/Config.xcconfig [hunk 1]
```diff
@@ -1,3 +1,3 @@
 TEAM_ID=
-BUNDLE_ID=com.myapplication.MyApplication
-APP_NAME=My application
+BUNDLE_ID=com.sciack.knews
```
Comment (chezori13) [inline iosApp/Configuration/Config.xcconfig]: android applicationId is such:
com.github.sciack.news
is it intentional

Comment (sciack) [reply iosApp/Configuration/Config.xcconfig]: forgot `github` :)

File: iosApp/Configuration/Config.xcconfig [hunk 2]
```diff
@@ -1,3 +1,3 @@
-TEAM_ID=
-BUNDLE_ID=com.myapplication.MyApplication
-APP_NAME=My application
+TEAM_ID=zuhlkers
+BUNDLE_ID=com.github.sciack.knews
```
Comment (chezori13) [inline iosApp/Configuration/Config.xcconfig]: hmm, knews or news ya

Comment (sciack) [reply iosApp/Configuration/Config.xcconfig]: :shit:

**Review by chezori13 — APPROVED** [review]

**Pull request merged**